### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by d-pozniak**
### HTML-CSS-Popup

[Demo](https://d-pozniak.github.io/html-css-popup/) |
[Code base](https://github.com/d-pozniak/html-css-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by d-pozniak** [comment]
I believe I`ve completed self-checks and fixed code accordingly.

File: submissions/d-pozniak/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,420 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>POPUP</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+</head>
+<body>
+<div class="header">
```
Comment (A-Ostrovnyy) [inline submissions/d-pozniak/html-css-popup/index.html:10]: div for page header? 
image

[header](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)

File: submissions/d-pozniak/html-css-popup/styles.css [hunk 2]
```diff
@@ -0,0 +1,210 @@
+@font-face {
+    font-family: Antic;
+    src: url(../fonts/AnticSlab-Regular.ttf);
```
Comment (A-Ostrovnyy) [inline submissions/d-pozniak/html-css-popup/styles.css:3]: Avoid using `.ttf` fonts for web, use `.woff2` + `.woff` instead.

File: submissions/d-pozniak/html-css-popup/styles.css [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    display: flex;
+    align-items: center;
+    margin: 0 15px;
+}
+.logo-img {
+    fill: #68838b;
+}
+.logo-title {
+    font-size: 54px;
+    font-family: Antic, Arial, sans-serif;
```
Comment (A-Ostrovnyy) [inline submissions/d-pozniak/html-css-popup/styles.css:53]: If you use this font for the whole page set this font for `html` or `body` elements.

File: submissions/d-pozniak/html-css-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,420 @@
...
+    <nav class="main-nav">
+        <div class="logo-wrapper">
+            <div class="logo-img">
```
Comment (A-Ostrovnyy) [inline submissions/d-pozniak/html-css-popup/index.html:13]: The logo should be a link and lead us to the home page.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome work!

**Comment by d-pozniak** [comment]
Thanks for your review, I`ll try to resolve issues as soon as possible.

File: submissions/d-pozniak/html-css-popup/styles.css [hunk 5]
```diff
@@ -0,0 +1,219 @@
+@font-face {
+    font-family: Antic;
+    src: url(../fonts/AnticSlab-Regular.woff),
+    url(../fonts/AnticSlab-Regular.woff2);
```
Comment (A-Ostrovnyy) [inline submissions/d-pozniak/html-css-popup/styles.css:4]: Use `.woff2` before `.woff`.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!
One last issue)

File: submissions/d-pozniak/html-css-popup/styles.css (hunk 5 above)
Comment (d-pozniak) [reply submissions/d-pozniak/html-css-popup/styles.css]: done :)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**